frontend/src/metabase/parameters/components/widgets/ParameterFieldWidget/normalizeValue.unit.spec.js

```diff
@@ -0,0 +1,26 @@
+import { normalizeValue } from "./normalizeValue";
+
+it("returns empty array if value is null", () => {
```

Comment:
```javascript
normalizeValue(undefined) // []
normalizeValue(true) // [true]
normalizeValue(false) // []
```

Apart from `false`, everything else seems to be working as expected?
Would you expect to see `[false]` as a result?

Reply:
Yes, great question @nemanjaglumac 

I would not expect to see [false] from how we consume this function today.

This is one of many ambiguities that TypeScript would remove: the type is a contract of what to expect, right?

Reply:
Exactly.
I could've re-framed the question. "Do we need to guard agains unexpected data type passed down to this function"?

Reply:
@nemanjaglumac I don't think so, as this has been used satisfactorily.